MonMaraicher/MapViewModel.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  MapViewModel.swift
+//  MonMaraicher
+//
+//  Created by Yves Charpentier on 24/02/2024.
+//
+
+import MapKit
+import SwiftUI
+
+final class MapViewModel: ObservableObject {
+    
+    @Published var userPosition: MapCameraPosition = .userLocation(fallback: .automatic)
+    
```

Comment:
Suggestion: spacing is not helping here, clear code is explicit but compact
Space can be used to split @Published with other property like let or init or func

Reply:
You mean the space between the class and @Published?